simulator/modules/garbage.py

```diff
@@ -1,96 +1,102 @@
-import simpy
+from utils import get_logger
```

Comment:
I am not a fan of utils package.  Suggestion: create a `log`  module.

Reply:
The util module have only three methods and each one has no relation with each other. So, you suggest to create a module for each method?

Reply:
Done. I have created log.py and config.py and removed the utils.py